src/pip/_internal/utils/misc.py

```diff
@@ -765,15 +765,33 @@ def unpack_file(
 
 
 def format_command_args(args):
-    # type: (List[str]) -> str
+    # type: (Union[List[str], List[Union[str, HiddenText]]]) -> str
```

Comment:
It looks like you have one too many `Union`s. Can't this just be `List[Union[str, HiddenText]]`?

Reply:
mypy doesn't allow it because this function is in use by [req_install.py](https://github.com/pypa/pip/blob/76a89542a8ea8b9ebbfe393f23ccbe764c727d3e/src/pip/_internal/req/req_install.py#L544) and a few other files.

mypy appears to treat `List[Union[str, HiddenText]]` as incompatible with `List[str]`.

Reply:
Okay, you might want to define a short-hand for `List[Union[str, HiddenText]]` then to make the type annotations more concise. `CommandArgs`? Then it can be `Union[List[str], CommandArgs]`.